src/project-tests/scatter-plot-tests.js

```diff
@@ -1,6 +1,109 @@
 import $ from 'jquery';
 import { testToolTip } from '../assets/globalD3Tests';
 
+//for given axis x or y, return width or height, begin and end coords, and arrays of tick elements and labels (for values)
+var thisAxis = function(x,y) {
+	var begin, end, axis;
+    if (x !== null) {
+	    axis = x;
+        //x-axis path x coordinates from d
+        //ex: M80.5,6V0.5H780.5V6 where 
+        //M establishes beginning coordinates of a line (80.5, 6). It should be the only comma so we can split there. We only need the x coordinate.
+        begin = axis.querySelector('path').getAttribute('d').split(',')[0].split('M')[1]
+        //H establishes a horizontal line from the last coordinate. That is the width of the axis.
+        //V means vertical lines. In this case, those demarcate line endings. We just need M and H.
+        end = axis.querySelector('path').getAttribute('d').split(',')[1].split('H')[1].split('V')[0]
+    } else {
+	    axis = y;
+	    begin = axis.querySelector('path').getAttribute('d').split(',')[1].split('H')[0];
+        end = axis.querySelector('path').getAttribute('d').split(',')[1].split('V')[1].split('H')[0];
+    }
+    return {
+        size: end - begin, 
+        begin: begin, 
+        end: end,
+        ticks: axis.querySelectorAll('.tick'),
+        text: axis.querySelectorAll('.tick text')
+    }
+}
+
+//for given tick, return value and x or y position
+var thisTick = function(x,y,i) {
+    if (x !== null) {
+        return {
+            text: x.querySelector('text').innerHTML,
+            //isolate x coordinate from "translate(x,y)"
+            px: x.getAttribute('transform').split(',')[0].split('(')[1]
+        }
+    } else {
+        return {
+	        text: y.querySelector('text').innerHTML,
+	        px: y.getAttribute('transform').split(',')[1].split(')')[1]
+        }
+    }
+}
+
+//for given feature, get values and coordinates from feature attributes
```

Comment:
In case this turns into a reusable function for other graphs, I've commented in an example.